app/controllers/dashboard_controller.rb

```diff
@@ -46,6 +46,11 @@ def edit_booking
     redirect_to root_path(date: booking_params[:booking_date])
   end
 
+  def delete_booking
+    @resource.destroy
```

Comment:
Not on my version of the code. My latest changes from yesterday fixes all issues and should not cause you any errors. Do a forced pull from your origin `git pull origin delete_booking -f` That should give you the latest changes without any merging conflicts. Make sure you run a local server and create a booking. After that you should be able to delete it.